test/e2e/prometheus_test.go

```diff
@@ -123,6 +123,13 @@ type remoteWriteTest struct {
 func TestPrometheusRemoteWrite(t *testing.T) {
 	ctx := context.Background()
 
+	// Get the Prometheus image to use, use the same than k8s's.
```

Comment:
nit: I am not sure if I understand the comment correctly

Reply:
adjusted it.